### PR TITLE
Add option to have same randomization for all users and sessions

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [x] tests added
- [x] commits conform to the [Commit message format](./CONTRIBUTING.md#commit)
- [x] documentation updated

**Squash commit message**

Add option to have same randomization for all users and sessions. This can be used for tests where a high cache hit rate is desired.

### CODE REVIEW

File: docs/settingup.md [hunk 1]
```diff
@@ -183,6 +183,9 @@ This section of the JSON file contains scheduler settings for the users in the l
   * `reuseUsers`: 
       * `true`: Every iteration for each concurrent user uses the same user and session.
       * `false`: Every iteration for each concurrent user uses a new user and session. The total number of users is the product of `concurrentusers` and `iterations`.
+  * `onlyinstanceseed`: Disable session part of randomization seed. Defaults to `false` if omitted.
```
Comment (carlcervin) [inline docs/settingup.md:186]: Add missing comma ("Defaults to 'false', if omitted.")

File: docs/settingup.md [hunk 2]
```diff
@@ -183,6 +183,9 @@ This section of the JSON file contains scheduler settings for the users in the l
...
+      * `true`: All users and sessions will have the same randomization sequence, only changing if `instance` flag is changed.
```
Comment (carlcervin) [inline docs/settingup.md:187]: Slight rewrite suggested:
"All users and sessions have the same randomization sequence, which only changes if the 'instance' flag is changed."

File: docs/settingup.md [hunk 3]
```diff
@@ -183,6 +183,9 @@ This section of the JSON file contains scheduler settings for the users in the l
...
+      * `false`: Normal randomization sequence, dependent on both `instance` parameter and current user session.
```
Comment (carlcervin) [inline docs/settingup.md:188]: Slight rewrite suggested:
"Normal randomization sequence, dependent on both the 'instance' parameter and the current user session."

File: generatedocs/data/params.json [hunk 4]
```diff
@@ -137,6 +137,11 @@
         "`true`: Every iteration for each concurrent user uses the same user and session.",
         "`false`: Every iteration for each concurrent user uses a new user and session. The total number of users is the product of `concurrentusers` and `iterations`."
     ],
+    "config.scheduler.settings.onlyinstanceseed": [
+        "Disable session part of randomization seed. Defaults to `false` if omitted.",
```
Comment (carlcervin) [inline generatedocs/data/params.json:141]: Slight rewrite suggested:
"Disable the session part of the randomization seed. Defaults to 'false', if omitted."

File: generatedocs/data/params.json [hunk 5]
```diff
@@ -137,6 +137,11 @@
...
+        "`true`: All users and sessions will have the same randomization sequence, only changing if `instance` flag is changed.",
```
Comment (carlcervin) [inline generatedocs/data/params.json:142]: See previous comment.

File: generatedocs/data/params.json [hunk 6]
```diff
@@ -137,6 +137,11 @@
...
+        "`false`: Normal randomization sequence, dependent on both `instance` parameter and current user session."
```
Comment (carlcervin) [inline generatedocs/data/params.json:143]: See previous comment.

File: generatedocs/generated/documentation.go [hunk 7]
```diff
@@ -216,6 +216,7 @@ var (
         "config.scheduler.settings.concurrentusers": { "Number of concurrent users to simulate. Allowed values are positive integers."  },  
         "config.scheduler.settings.executiontime": { "Test execution time (seconds). The sessions are disconnected when the specified time has elapsed. Allowed values are positive integers. `-1` means an infinite execution time."  },  
         "config.scheduler.settings.iterations": { "Number of iterations for each 'concurrent' user to repeat. Allowed values are positive integers. `-1` means an infinite number of iterations."  },  
+        "config.scheduler.settings.onlyinstanceseed": { "Disable session part of randomization seed. Defaults to `false` if omitted.","`true`: All users and sessions will have the same randomization sequence, only changing if `instance` flag is changed.","`false`: Normal randomization sequence, dependent on both `instance` parameter and current user session."  },
```
Comment (carlcervin) [inline generatedocs/generated/documentation.go:219]: I guess this test is automatically generated and therefore does not need any review?

File: scheduler/simple_test.go [hunk 8]
```diff
@@ -46,3 +54,94 @@ func TestSimpleSched(t *testing.T) {
...
+
+func (settings TestOnlyInstanceSeedAction) Validate() error {
+	return nil
+}
+func (settings *TestOnlyInstanceSeedAction) Execute(sessionState *session.State,
+	actionState *action.State, connectionSettings *connection.ConnectionSettings, label string, reset func()) {
+	settings.Result = append(settings.Result, sessionState.Randomizer().Rand(100))
+}
+
+func TestOnlyInstanceSeed(t *testing.T) {
```
Comment (SeBBBe) [inline scheduler/simple_test.go:70]: Nice test case!

File: generatedocs/generated/documentation.go (hunk 7 above)
Comment (DnlLrssn) [reply generatedocs/generated/documentation.go:219]: both settingup.md and documentation.go are generated from params.json, so only need to comment on params.json

**Comment by DnlLrssn** [comment]
> 
> 
> Looks good!
> Noticed that I accidently changed ` to ' when it comes to the instance flag... Please change it back, if possible.

ok fixed, docs re-generated now also.

**Pull request merged**